Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1268,11 +1264,7 @@ - (void)processNextQueueItem {
                 [req processResponse:nil error:[NSError errorWithDomain:BNCErrorDomain code:BNCInitError userInfo:@{ NSLocalizedDescriptionKey: @"Branch User Session has not been initialized" }]];
                 return;
             }
-            
-            if (![req isKindOfClass:[BranchCloseRequest class]]) {
-                [self clearTimer];
-            }
-            
+                        
```

Comment:
Going through this PR again and it seems like you didn't address this comment? Are you still working on this?